lib/Runtime/Language/Chakra.Runtime.Language.vcxproj

```diff
@@ -262,9 +262,9 @@
     <ARMASM Include="$(MSBuildThisFileDirectory)arm64\arm64_CallEhFrame.asm">
       <ExcludedFromBuild Condition="'$(Platform)'!='Arm64'">true</ExcludedFromBuild>
     </ARMASM>
-    <None Include="amd64\stackframe.inl" />
-    <None Include="AsmJSEncoder.inl" />
-    <None Include="AsmJSEncoderHandler.inl" />
+    <None Include="amd64\StackFrame.inl" />
```

Comment:
> Should you change this backslash to a forward slash? Is that even allowed in a vcxproj?

Not sure. Since vcxproj is intended to be managed by VS and use back slash, I didn't change it.